src/index.md

```diff
@@ -333,6 +333,139 @@ For the purposes of brevity this document will use the term "bug" to refer to bo
 
 _Congratulations!_ Your changeset will now make its way towards a promoted build. When the changeset becomes part of a promoted build, the bug's "Resolved in Build" will have a value of \"b\[1-9\]\[0-9\]&ast;\" to indicate the build number.
 
+# Testing Changes
+
+In addition to your own Java applications, OpenJDK have support for two test frameworks, JTReg and GTest. JTReg is a Java regression test framework that is used for most of the tests that are included in the OpenJDK source repository. The Google Test (GTest) framework is intended for unit testing of the C++ native code. Currently only JVM testing is supported by the GTest framework. Other areas use JTReg for unit testing of C++ code.
+
+This section provides a brief summary of how to get started with testing in OpenJDK. For more information on configuration and how to use the OpenJDK test framework, a.k.a. "run-test framework", see [`doc/testing.md`](https://htmlpreview.github.io/?https://github.com/openjdk/jdk/blob/master/doc/testing.html).
+
+In general all changes should come with a regression test so if you're writing product code you should also be writing test code. There are a few examples where it doesn't make sence to write an explicit regression test. These should be tagged in JBS with one of the [noreg-labels](#noreg).
+
+A few key items to think about when writing a regression test:
+
+* A regression test should execute fast - a few seconds at most
+* The test should only test the desired functionality - if you have several features to test, write more tests
+* The test should pass reliably on all supported platforms - watch out for platform-specific differences such as path separators
+
+The JTReg documentation has a section on [how to write good JTReg tests](https://openjdk.java.net/jtreg/writetests.html).
+
+## JTReg
+
+In-depth documentation about the JTReg framework is found here: [JTReg harness](https://openjdk.java.net/jtreg/). JTReg itself is available in the [Code Tools Project](https://openjdk.java.net/projects/code-tools/).
+
+Below is a small example of a JTReg test. It’s a clean Java class with a main method that is called from the test harness. If the test fails we throw a RuntimeException. This is picked up by the harness and is reported as a test failure. Try to always write a meaningful message in the exception. One that actually helps with understanding what went wrong once the test fails.
+
+    /*
+     * @test
+     * @summary Make sure feature X handles Y correctly
+     * @run main TestXY
+     */
+    public class TestXY {
+        public static void main(String[] args) throws Exception {
+            var result = X.y();
+            if (result != expected_result) {
+                throw new RuntimeException("X.y() gave " + result + ", expexted " + expected_result);
+            }
+        }
+    }
```

Comment:
Should there be a mention of trying to be consistent with other tests in a test folder, that is if they are using TestNG,  follow that pattern

Should we indicate when to use a TEST.properties file vs. including the properties within the test?

Reply:
>Should we indicate when to use a TEST.properties file vs. including the properties within the test?

`TEST.properties` files aren't intended to be used by tests, they are jtreg configurations file, so using them for test-specific properties might lead to name clashes and unexpected behaviors.

Reply:
Let's agree on not adding anything about TEST.properties at this point. I have added a note in the todo-list about the need for documented best practices for this. Once the text is written we can discuss where to put it.